jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -152,6 +245,9 @@ YUI.add('user-profile', function() {
 
     render: function() {
       var whitelist = ['path', 'name', 'user', 'uuid', 'host-ports'];
+      var charmCount = this.state.charmList && this.state.charmList.length;
```

Comment:
To help with minifcation here... could you...
`var charmList = this.state.charmList`...